actionview/lib/action_view/template.rb

```diff
@@ -122,10 +122,10 @@ def self.finalize_compiled_template_methods=(_)
 
     extend Template::Handlers
 
-    attr_reader :source, :identifier, :handler, :original_encoding, :updated_at
+    attr_reader :source, :identifier, :handler, :original_encoding
```

Comment:
`:update_at` had exist for about 9 years. 38d78f99d52801d8392a7229b40edae74cc3d142
Should it be deprecated first like `:original_encoding`? 0e13941dbe9c4840175e5c6f5b2c3f572249a4ec

Reply:
Yes, we should.

Reply:
on it!